rplugin/python3/docstring.py

```diff
@@ -127,7 +127,7 @@ def generate_docstring(self):
         current_line_number = int(current_line_number) - 1
         current_line = self.nvim.current.line
 
-        if not current_line.startswith("def "):
```

Comment:
In the case below, `current_line.startswith("def ")` is `false` because `current_line` starts with `<space>`.

```python
# ...
    def main(arg):
        something
# ...
```